### PR TITLE
KEYCLOAK-3161 KEYCLOAK-3165 Migrate SAML tests from old testsuite to …

### PR DESCRIPTION
**Pull request by mhajas**
…integration-arquillian

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/SamlClient.java [hunk 1]
```diff
@@ -121,12 +121,17 @@ public HttpPost createSamlUnsignedResponse(URI samlEndpoint, String relayState,
...
+                return null;
+            }
+
             @Override
             public HttpPost createSamlSignedRequest(URI samlEndpoint, String relayState, Document samlRequest, String realmPrivateKey, String realmPublicKey) {
                 return createSamlPostMessage(samlEndpoint, relayState, samlRequest, GeneralConstants.SAML_REQUEST_KEY, realmPrivateKey, realmPublicKey);
             }
 
-            private HttpPost createSamlPostMessage(URI samlEndpoint, String relayState, Document samlRequest, String messageType, String privateKeyStr, String publicKeyStr) {
+            private HttpPost createSamlPostMessage(URI samlEndpoint, String relayState, Document samlRequest, String messageType, String realmPrivateKeyStr, String realmPublicKey) {
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/SamlClient.java]: why is this renamed? It needn't be any key related to a realm

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Almost approved, just small nit re parameter naming.

File: testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/SamlClient.java (hunk 1 above)
Comment (mhajas) [reply testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/SamlClient.java]: Sorry I thought this method is overwriting the one in parent class which does have arguments with realm* prefix. I will revert it.

**Comment by stianst** [comment]
@mhajas bump

**Comment by mhajas** [comment]
Sorry, I thought Hynek will get a notification after my new commit. I removed realm prefix in my last change, so I think this is done.

**Review by hmlnarik — APPROVED** [review]
